pnc_cli/buildrecords.py

```diff
@@ -71,11 +71,14 @@ def get_build_record(id):
 @arg("-p", "--page-size", help="Limit the amount of Artifacts returned", type=int)
 @arg("-s", "--sort", help="Sorting RSQL")
 @arg("-q", help="RSQL query")
-def list_built_artifacts(id, page_size=200, sort="", q=""):
+@arg("--page-index", help="Page index")
```

Comment:
maybe specify the `type=int` here for some validation checking.

Reply:
Good idea, added.